pkg/reconciler/brokercell/resources/deployments.go

```diff
@@ -34,6 +35,20 @@ func MakeIngressDeployment(args IngressArgs) *appsv1.Deployment {
 	// Decorate the container template with ingress port.
 	container.Env = append(container.Env, corev1.EnvVar{Name: "PORT", Value: strconv.Itoa(args.Port)})
 	container.Ports = append(container.Ports, corev1.ContainerPort{Name: "http", ContainerPort: int32(args.Port)})
+	container.LivenessProbe = &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:   "/healthz",
+				Port:   intstr.FromInt(args.Port),
+				Scheme: corev1.URISchemeHTTP,
+			},
+		},
+		FailureThreshold:    3,
```

Comment:
Where did these numbers come from?

Reply:
Used the same from Eventing core broker.

Reply:
I recall I had some problems with these numbers on the old implementation... https://github.com/knative/eventing/issues/2060

Reply:
Ack. The fact that Eventing core hasn't fixed the issue might be an indicator that this is not a common problem? We can always adjust these numbers if we find problem.